README.md

```diff
@@ -104,7 +113,7 @@ export PATH=$PATH:$ANDROID_HOME/tools/bin
 export PATH=$PATH:$ANDROID_HOME/platform-tools
 ```
 
-**Troubleshootings:**
+**Troubleshooting:**
```

Comment:
We should point to the Babylon Native troubleshooting as well, including the stuff about Android emulators (we may even have an issue tracking this).

Reply:
Oh yeah, I forgot to add this.

Reply:
I added two tips, one pointing to Android Emulator directly and another for general Babylon Native docs.